drivers/serial/uart_rpi_pico.c

```diff
@@ -53,27 +66,238 @@ static int uart_rpi_init(const struct device *dev)
 		return -EINVAL;
 	}
 
+	hw_clear_bits(&uart_get_hw(uart_inst)->lcr_h, UART_UARTLCR_H_FEN_BITS);
+	uart_get_hw(uart_inst)->dr = 0U;
+
+#ifdef CONFIG_UART_INTERRUPT_DRIVEN
+	config->irq_config_func(dev);
+#endif /* CONFIG_UART_INTERRUPT_DRIVEN */
+
 	return 0;
 }
 
+static int uart_rpi_err_check(const struct device *dev)
+{
+	int errors = 0;
+
+	const struct uart_rpi_config * const config = dev->config;
```

Comment:
Please use reverse christmas tree declaration.
And no space between declarations.

Reply:
Done!